fossilize.cpp

```diff
@@ -995,6 +999,24 @@ static void hash_pnext_struct(const StateRecorder *,
 	h.u32(info.dstAccessMask);
 }
 
+static void hash_pnext_struct(const StateRecorder *,
+                              Hasher &h,
+                              const VkPipelineFragmentShadingRateStateCreateInfoKHR &info,
+			      const DynamicStateInfo *dynamic_state_info)
+{
+	if (dynamic_state_info && !dynamic_state_info->fragment_shading_rate)
+	{
+		h.u32(info.fragmentSize.width);
+		h.u32(info.fragmentSize.height);
+		h.u32(info.combinerOps[0]);
+		h.u32(info.combinerOps[1]);
+	}
+	else
+	{
+		h.u32(0);
```

Comment:
Don't need to hash in 0 here. sType is already hashed in automatically.

Reply:
Fixed.